### PR TITLE
Add HSV-sorted color table to docs

### PR DESCRIPTION
**Pull request by user27182**
### Overview

Show colors sorted by HSV in a separate dropdown. This presents an alternative way of viewing the colors. The sorted is the same as used in matplotlib here https://matplotlib.org/stable/gallery/color/named_colors.html#css-colors

### CODE REVIEW

File: doc/source/make_tables.py [hunk 1]
```diff
@@ -2067,3 +2095,8 @@ def make_all_tables():
     # Make dataset gallery carousels
     os.makedirs(DATASET_GALLERY_DIR, exist_ok=True)
     make_all_carousels(CAROUSEL_LIST)
+
+
+if __name__ == '__main__':
+    ColorTable.generate()
+    SortedColorTable.generate()
```
Comment (user27182) [inline doc/source/make_tables.py:2102]: For debugging only, remove later

File: pyvista/plotting/colors.py [hunk 2]
```diff
@@ -697,9 +697,15 @@ class Color:
     The internally used representation is an integer RGBA sequence (values
     between 0 and 255). This might however change in future releases.
 
-    Refer to the table below for a list of supported colors
+    Refer to the tables below for a list of supported colors.
 
-    .. include:: ../color_table/color_table.rst
+    .. dropdown:: Colors Sorted By Name
```
Comment (tkoyama010) [inline pyvista/plotting/colors.py:702]:
```suggestion
    .. dropdown:: Colors Sorted by Name
```

File: pyvista/plotting/colors.py [hunk 3]
```diff
@@ -697,9 +697,15 @@ class Color:
...
+
+        .. include:: ../color_table/color_table.rst
+
+    .. dropdown:: Colors Sorted By Hue, Saturation, and Value
```
Comment (tkoyama010) [inline pyvista/plotting/colors.py:706]:
```suggestion
    .. dropdown:: Colors Sorted by Hue, Saturation, and Value
```

File: pyvista/plotting/colors.py [hunk 4]
```diff
@@ -1057,17 +1063,21 @@ def hex_rgb(self) -> str:  # numpydoc ignore=RT01
     def name(self) -> str | None:  # numpydoc ignore=RT01
         """Get the color name.
 
+        Refer to the tables below for a list of supported colors.
+
+        .. dropdown:: Colors Sorted By Name
```
Comment (tkoyama010) [inline pyvista/plotting/colors.py:1068]:
```suggestion
        .. dropdown:: Colors Sorted by Name
```

File: pyvista/plotting/colors.py [hunk 5]
```diff
@@ -1057,17 +1063,21 @@ def hex_rgb(self) -> str:  # numpydoc ignore=RT01
...
+
+            .. include:: ../color_table/color_table.rst
+
+        .. dropdown:: Colors Sorted By Hue, Saturation, and Value
```
Comment (tkoyama010) [inline pyvista/plotting/colors.py:1072]:
```suggestion
        .. dropdown:: Colors Sorted by Hue, Saturation, and Value
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

**Review by tkoyama010 — DISMISSED** [review]
LGTM

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**